blog/_posts/2025-04-22-Enhance-security-OS-Security-Patch.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Enhancing Your Operating System's Security with OS Security Patches in AKS"
+description: "Learn the what, the whys, and the hows of configuring OS Security Patch Auto upgrade channel. The article also covers some reasons for OS Security Patch, with some primary benefits being increased performance, better security, and minimal disruption to workloads."
+date: 2025-04-22
+authors:
+- Kaarthikeyan Subramanian
+- Ye Wang
+categories: 
+- Security
+- Workload SLO
+- Upgrades
+---
+
+
+## Why do we need Security patches at the Node Operating System level?
+
+Operating System (OS) security patches are crucial for maintaining the integrity and security of your systems. These patches address vulnerabilities that could be exploited by malicious actors and ensure that your system remains protected against the latest threats. Regularly applying OS security patches helps to safeguard sensitive data, maintain compliance with security standards, and prevent potential disruptions caused by security breaches. At Azure Kubernetes Service we are constantly working hard and innovating to guarantee maximum security for your Node operating systems. 
+Presently there are two ways to apply security patches at the OS level automatically with AKS. Read more [OS Security Patch](https://learn.microsoft.com/azure/aks/auto-upgrade-node-os-image?tabs=azure-cli)
+
+**Auto Node Image Channel** - AKS updates nodes weekly with a newly patched VHD for security and bug fixes. This update, which follows maintenance windows and surge settings, incurs no extra VHD cost. Linux unattended upgrades are disabled by default when using this channel. Node image upgrades are supported as long as the cluster's Kubernetes minor version is in support. These AKS-tested node images are fully managed and applied with safe deployment practices.
+**OS Security Patch Channel** - Several customers may need only the security packages for their OS without additional bug fixes and updates. Customers previously relied on nightly updates and reboot mechanisms like KURED for security updates. Past outages from untested packages highlighted the need for a managed security patch-only channel at the node level.  With OS Security Patch channel you get precisely that a fully managed attended Node OS security only channel! This automates manual processes, eliminating the need for tools like KURED. The Security-Patch channel reimages nodes only when necessary and provides live patch updates with zero disruption, respecting planned maintenance windows.
+
+## OS Security Patch channel - How it fares against Node Image Channel
+When comparing OS security patch channel with the Node Image channel, several advantages become apparent:
+
+- **Faster Patching Cadence**: OS security patches are released at a much faster rate than Node Image updates on average 1 to 2 weeks ahead. This ensures that critical vulnerabilities (CVEs) are addressed promptly, reducing the window of exposure. 
```

Comment:
We've gotten quite a few questions about CVEs. Would it be useful to also link to some of our new  releases on here to? https://learn.microsoft.com/en-us/azure/aks/security-bulletins/overview?tabs=aks-addons%2Caks-cluster%2Caks-node-image https://releases.aks.azure.com/webpage/index.html?

Reply:
Yes i can throw in the link.